src/_product-updates/2024-08-10-feedbacks-gathering.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Documentation feedback 
+tags: [New]
+image: /images/changelog/feedback.png
+---
+
+![feedback.png](/images/changelog/feedback.png)
+
+Feedback from real-life users is the best way to improve your documentation. Our last update got you covered! 
```

Comment:
In the Product Updates, I prefer to maintain a more informative/educational tone and moderate enthusiasm (which I save more for social blasts), unless announcing an absolutely incredible or major feature.

Reply:
So maybe remove this part, and start with "Say hi to the new "Give feedback" button, designed to stand out from the rest of the topbar." ?

Reply:
I reworked it, will be updated with the next commit 🙏